app/config/Module.scala

```diff
@@ -25,6 +25,7 @@ class Module() extends AbstractModule {
   override def configure(): Unit = {
     bind(classOf[UploadProgressTracker]).to(classOf[MongoBackedUploadProgressTracker])
     bind(classOf[SaxParser]).to(classOf[MDRSchemaValidatingParser])
+    ()
```

Comment:
empty bracket?

Reply:
Mark did that as part of turning on the fatal warnings flag, I assume it's to force a specific return type of Unit to stop the compilation failing with the new flag turned on. Is that right @ers2015112 ?